_posts/2020-08-11-backports-application.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: From zero to hero on backporting pull requests
+date: 2020-08-11
+tags: user-story
+author: gastaldi
+---
+= From zero to hero on backporting pull requests
+
+Before we dig into the solution, let me give you a quick explanation about our release process.
+
+== Release process
+
+=== Minor versions
+Our master branch is always ready for the next minor release. That means that any pull requests merged in there are available in the next major release. This process is usually very smooth and involves no backporting.
+
+=== Patch releases or CRs
+Whenever a new patched version (or a second CR) is about to be released in Quarkus, our Release engineer starts backporting commits from pull requests that were merged in the master branch.
+How do they know which pull request to grab? We have a `triage/backport?` label that our team add to pull requests for features that would be worth having it in the upcoming patch release.
+
+== What the app does?
```

Comment:
All of a sudden, you start talking about an app, assuming that I (as a reader) know it exists and am interested in it.

How about saying something like: "Considering the amount of backports, this gets tedious and error prone when done manually. So we decided to automate that, and built an application. Of course with Quarkus!"